README.md

```diff
@@ -86,20 +86,18 @@ By default, Dagster runs the pipeline in a subprocess on the same computer it ru
 
 ### Run OAI Cartilage Thickness Pipeline
 
-With Dagster running (`pixi run dev`), create a `patient_ids.json` file in the `knee-sarg/data/oai-sampler` directory. The JSON file should contain an array of OAI patient IDs. Example:
+With Dagster running (`pixi run dev`), create a `study_uids_to_run.json` file in the `knee-sarg/data/oai-sampler` directory.
```

Comment:
Does this mean that `patient_ids.json` is now ignored? If so, can laterality(left/right) column be added to `study_uid_to_vol_path.csv`? The `study_description` column contains a combination of information.

Reply:
patient_ids is still active, but trigging runs by materialzing the `oai_study_uids_to_run` asset make more sense to me.

I will add laterality to the csv.  Why do you want it?  Will you use the CSV to trigger your own calls to your OAI pipeline?

Reply:
I would leave description of both `patient_ids` and `oai_study_uids_to_run`. Adding laterality column allows that CSV to be sorted/filtered by that attribute. That would be important if `patient_ids` trigger was dropped, but is less important otherwise.

Reply:
In `make_study_uids_json.py`, I added some example Pandas API calls that filters  study_uid_to_vol_path.csv by laterality and patient ID
https://github.com/PaulHax/knee-sarg/compare/c97795e808fbe79229dd432b24fb23b1dea4cb1b..2928893d7e846f69358ec61ccd04a8078e37c424